scss/content.scss

```diff
@@ -0,0 +1,4 @@
+.content{
+  padding-top: 100px;
```

Comment:
remember what we talk about the paddings. they are usually the same both sides

i would recommend to do it like 
```
padding: 100px 150px;
```